packages/web/src/js/Toast.ts

```diff
@@ -7,6 +7,8 @@ import {
   ATTRIBUTE_DATA_POPULATE_FIELD,
   ATTRIBUTE_DATA_SNIPPET,
   ATTRIBUTE_DATA_TARGET,
+  CLASS_LINK_DISABLED,
```

Comment:
I think this should be CLASS_NAME_LINK_DISABLED. But 🤷 

(and if you change it, the second one should be `CLASS_NAME_LINK_UNDERLINED`)